docs/source/swift-scripting.md

```diff
@@ -3,7 +3,9 @@ title: Swift scripting
 sidebar_title: Swift scripting
 ---
 
-Apollo Client for iOS enables you to use Swift scripting to perform certain operations that otherwise require the command line. This document guides you through setting up a Swift Package Manager executable and then using it to: 
+Apollo Client for iOS enables you to use Swift scripting to perform certain operations that otherwise require the command line. 
```

Comment:
I think this could be clearer about the use cases. I was under the impression that this was just a replacement for the schema download and code generations commands. But only after talking to @designatednerd offline found out that it actually enables you to do a lot more complex custom scripting.

Doesn't have to be in the PR, but maybe an example of how you would write a custom script for a contrived use case would help illustrate this point!

Reply:
🤔 I mean, we are essentially using an example of this in the `SwiftScripts` in this repo in order to generate our docs. Is that what you're thinking of, or are you thinking of something else?

Reply:
Kind of! Looking at that example, the only part of the Codegen lib we are using is the `CodegenLogger`, so not a really great example of the power that this provides, but it shows the basic concept.

Reply:
I also imagine that the scripting tools could get more powerful in the future if we provide a way to introspect on the compiled IR.

Reply:
ah i mean that as more of an example of "here are other things you can do with Swift scripting" than what you can do specifically with the codegen lib